app/tests/spec/views/choose_what_to_sync.js

```diff
@@ -121,6 +130,26 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('session verification polling', () => {
```

Comment:
No biggie obviously, but if you're doing all the setup inside the test itself, does the `describe` here have a point?